10_code/NC_covid_response_analysis.py

```diff
@@ -0,0 +1,37 @@
+import pandas as pd
+import datetime
+
+daily_cases = pd.read_csv("./data/us-counties-covid-cases.csv")
+daily_cases = daily_cases[daily_cases["state"] == "North Carolina"]
+daily_cases.head()
+
+# set comparison period to be 14 days
+numdays = 14
```

Comment:
Could you elaborate why choosing a 14-day window? My concern would be that 14-day is still within the latency period for the virus to show clinical symptoms and thus offers an underestimate of the overall case trend.

Reply:
Changed analysis to 3 weeks before and after the policy announcement since the incubation period of COVID is approximately 2 weeks. Also, Mecklenburg county only had covid cases data from 3/12 so only about 2.5 weeks of data were captured in the pre-policy analysis for this county, which we will address as a limitation in our report.

Reply:
Awesome! Will note that in the report writing.